### ISSUE TITLE
How do you toggle dark mode?

### ISSUE THREAD

**Issue by GreenAppers** [LanikSJ/slack-dark-mode#153]
**Describe the bug**
I can't figure out how to toggle dark mode.  I'd like light mode during day and dark mode at night.  Either needs more docs or the feature itself.

**To Reproduce**
Try to toggle dark mode.

**Expected behavior**
Clear option for toggling dark mode.

**Comment by krazyito65** [comment]
Right, we haven't updated the docs for light mode yet.

Basically, once you install you can just use the 'update feature' 

Mac/Linux:
```bash
# Dark
./slack-dark-mode.sh -u # u stands for update CSS (so just apply it)
# Light
./slack-dark-mode.sh -l # l stands for light mode
```

Windows:
```powershell
# Dark
.\slack-dark-mode.ps1 -UpdateOnly
# Light
.\slack-dark-mode.ps1 -LightMode
```

**Comment by krazyito65** [comment]
Probably should keep this open as an issue for updating the Docs.

**Comment by LanikSJ** [comment]
Fair enough.

**Comment by LanikSJ** [comment]
This was addressed in #152
